opm/input/eclipse/EclipseState/Grid/Carfin.cpp

```diff
@@ -110,6 +110,8 @@ namespace Opm
         std::string name = "LGR";
         default_count += update_default_name(deckRecord.getItem<ParserKeywords::CARFIN::NAME>(), name);      
 
+        std::string parent_name = "";
```

Comment:
strings are empty by default.

Reply:
I am setting the default as "GLOBAL".

Reply:
uhm, no, you are initializing to "" - the empty string, which is unnecessary as a string is empty by default.

Reply:
Sorry, this is the new commit I am about to push.